storm/src/main/storm/mesos/util/PrettyProtobuf.java

```diff
@@ -223,7 +223,7 @@ private static String setToString(Set set) {
           break;
         default:
           // If hit, then a new Resource Type needs to be handled here.
-          value = "Unrecognized Resource Type: `" + r.getType() + "'";
+          value = String.format("Unrecognized Resource Type: `%s'", r.getType());
```

Comment:
Yup, this was intentional on my part, following a somewhat dubious convention that exists throughout unix for printing string keywords:
* https://english.stackexchange.com/questions/17695/any-reference-on-the-usage-of-a-backtick-and-single-quotation-mark-like-this/18669#18669
* https://emacs.stackexchange.com/questions/9881/how-to-quote-symbols-in-a-documentation-string